add CfA quiz instructions (#246)
UBC-MDS/UBC-MDS.github.io: resources_pages/quiz_cfa.md

---
layout: page
title: MDS quizzes - CfA
---

> 

Reviewer: ```suggestion
## What is CfA?

The CfA stands for the Centre for Accessibility. It provides disability-related accommodations and programming designed to remove barriers for students with disabilities or ongoing medical conditions in all aspects of university life. For more information, visit the [Centre for Accessibility](https://students.ubc.ca/about-student-services/centre-for-accessibility).

In the MDS program, if you have a disability or ongoing medical condition, you can request quiz accommodations through the CfA.
```

Reviewer: add a brief intro to CfA in the beginning

## Accommodations

> If you have an accommodation letter from the Center for Accessibility, please send them to the course coordinators to ensure you receive your approved accommodations throughout the program.

Reviewer: ```suggestion
If you have an accommodation letter from the Center for Accessibility (CfA), please send it to the course coordinators to ensure that your approved accommodations are provided throughout the program.
```

> If **all** of your accommodations **can** be accommodated by the CBTF (check the list [here](https://cbtf.ubc.ca/students/accommodations)), you will still write your quizzes at the CBTF. Please follow instructions on [MDS quizzes](https://ubc-mds.github.io/resources_pages/quiz/).

Reviewer: ```suggestion
If **all** of your accommodations **can** be met by the CBTF (check the list [here](https://cbtf.ubc.ca/students/accommodations)), you will continue to write your quizzes at the CBTF. Please follow the instructions on the [MDS quizzes](https://ubc-mds.github.io/resources_pages/quiz/) page to register for your quizzes.
```

> Otherwise, please follow the instructions below.

Reviewer: ```suggestion
For accommodations that cannot be provided by the CBTF, follow the instructions below.
```

> ## List of MDS Quiz instructions that diverge

Reviewer: Combined this section to the sentence ^^^

You should still go through the [MDS quizzes](https://ubc-mds.github.io/resources_pages/quiz/) page. The only sections that do not apply to you is:
- [PrairieTest and Making a Reservation for your Quiz](https://ubc-mds.github.io/resources_pages/quiz/#prairietest-and-making-a-reservation-for-your-quiz)
- [Quiz Room Procedures](https://ubc-mds.github.io/resources_pages/quiz/#quiz-room-procedures)

## CfA Quiz Booking

> - On [PrairieTest](https://us.prairietest.com/pt), you do **not** need to make a reservation. Instead, we will assign you to a special session allowing you to write your quiz outside of the CBTF.

Reviewer: ```suggestion
You do **not** need to make a reservation on [PrairieTest](https://us.prairietest.com/pt). Instead, we will assign you to a special session to write your quiz outside of the CBTF.
```

> - You should book each of your quizzes with the CfA. Instructions are provided by [the CfA](https://students.ubc.ca/about-student-services/centre-for-accessibility).

Reviewer: ```suggestion
- Book each of your quizzes through the CfA following their instructions [here](https://students.ubc.ca/about-student-services/centre-for-accessibility).
```

>     - You can pick your own quiz times, as long as they fall in the quiz window specified in the calendar, **and** they cannot happen during your lecture and lab times.

Reviewer: ```suggestion
    - You can choose your quiz times as long as they fall within the quiz window specified in the calendar **and** do not overlap with your lecture or lab times.
```
